packages/toast/toaster/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Toaster as ReactHotToaster } from 'react-hot-toast'
+
+export default function Toaster() {
+  return (
+    <ReactHotToaster
+      position="bottom-left"
+      gutter={16}
+      containerStyle={{
+        inset: 24,
+      }}
+      toastOptions={{
+        style: {
+          margin: 0,
+          padding: 0,
+          maxWidth: 404,
+          backgroundColor: 'transparent',
+          boxShadow: 'none',
+          borderRadius: 0,
```

Comment:
Unsetting some styling coming from react-hot-toast. This styling is defined within the Notification components.

Reply:
nit: A comment here saying this could be helpful for the future!